examples/webgl_math_orientation_transform.html

```diff
@@ -46,29 +47,40 @@
 
 		var camera, scene, renderer, mesh, target;
 
-		var spherical = new THREE.Spherical();
-		var rotationMatrix = new THREE.Matrix4();
-		var targetRotation = new THREE.Quaternion();
+		var tempObject = new THREE.Object3D();
+		var targetQuaternion = new THREE.Quaternion();
 		var clock = new THREE.Clock();
-		var speed = 2;
+		var speed = 2; // radians per second
 
 		init();
 		animate();
 
 		function init() {
 
+			renderer = new THREE.WebGLRenderer( { antialias: true } );
+			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( window.innerWidth, window.innerHeight );
+			document.body.appendChild( renderer.domElement );
+
+			scene = new THREE.Scene();
+
 			camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 0.01, 10 );
 			camera.position.z = 5;
 
-			scene = new THREE.Scene();
+			var controls = new THREE.OrbitControls( camera, renderer.domElement );
+			controls.maxDistance = 8;
+			controls.minDistance = 2;
+			controls.enablePan = false;
 
-			var geometry = new THREE.ConeBufferGeometry( 0.1, 0.5, 8 );
-			geometry.rotateX( Math.PI * 0.5 );
+			var geometry = new THREE.BoxBufferGeometry( 1, 0.5, .1 );
 			var material = new THREE.MeshNormalMaterial();
 
 			mesh = new THREE.Mesh( geometry, material );
+			//mesh.position.y = 1;
 			scene.add( mesh );
 
+			mesh.add( new THREE.AxesHelper( 1 ) );
```

Comment:
When using the cone, this is not necessary.

Reply:
The axes illustrate what `lookAt()` is doing. An orientation is more than a direction.